packages/aws-cdk-lib/core/lib/schedule.ts

```diff
@@ -0,0 +1,205 @@
+import { Construct } from 'constructs';
+import { Annotations } from './annotations';
+import { Duration } from './duration';
+import { TimeZone } from './time-zone';
+
+/**
+ * A Core Schedule. This construct is not meant to be used as is or exposed to consumers in other modules.
+ * It is meant to be extended by other modules that require some sort of schedule implementation. All
+ * methods in `core.Schedule` are protected, so that construct authors can decide which APIs to expose.
+ *
+ * @see https://docs.aws.amazon.com/eventbridge/latest/userguide/scheduled-events.html
+ */
+export abstract class Schedule {
+  /**
+   * Construct a one-time schedule from a date.
+   *
+   * @param date The date and time to use. The millisecond part will be ignored.
+   * @param timeZone The time zone to use for interpreting the date. Default: - UTC
+   */
+  protected static protectedAt(date: Date, timeZone?: TimeZone): Schedule {
```

Comment:
did some research and it seems like there's nothing inherently wrong about having a `protected static`, just that the need for such a thing is incredibly niche. I believe this is one such scenario where this makes sense, because we want to show construct authors the available APIs that make sense, but we don't want to expose them as is to consumers